common/rfb/CSecurityTLS.cxx

```diff
@@ -385,14 +384,14 @@ void CSecurityTLS::checkSession()
 
   /* Certificate has some user overridable problems, so TOFU time */
 
-  homeDir = os::getvnchomedir();
-  if (homeDir == NULL) {
-    throw AuthFailureException("Could not obtain VNC home directory "
+  hostsDir = os::getvncconfigdir();
```

Comment:
As mentioned in the other comment, I think this fits better in some "data" directory.